apps/srp.c

```diff
@@ -553,6 +553,8 @@ int srp_main(int argc, char **argv)
             user = *(argv++);
         else {
             user = NULL;
+            if (mode == OPT_LIST)
+                break;
```

Comment:
"Required" is a tricky term in the context. Is minimizing confusion and simplification required? It's nice to have, but it's not actually required.